index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Credentials API
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class="remove">
+    "use strict";
+    // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+    var respecConfig = {
+      github: "WICG/digital-credentials",
+      editors: [
+        {
+          name: "Marcos Caceres",
+          email: "marcosc@apple.com",
+          company: "Apple Inc.",
+          companyURL: "https://apple.com",
+        },
+        // Add additional editors here.
```

Comment:
Can we have someone from Chrome here (you can use me as a placeholder, we'll find someone to edit this more long term) so that we can make sure that what's in the spec also represents what we'd implement in Chrome.

Also, procedurally, can we agree on blocking normative PR merges with an LGTM from the two implementions?

It would be good if we could also get Ben from Mozilla here to help edit, in case Firefox has an immediate desire to implement this too.

Reply:
Yeah, OWNERs could help here too.

Reply:
not sure how to parse `can we agree on blocking normative PR merges with an LGTM from the two implementions`. Do you intend to merge a PR if it has approvals from at least 2 implementors regardless of the number of people requesting for changes?

Reply:
thats how I read it, you would just set implementation reps as code owners, and require approval from them.

Reply:
Yeah, basically. That would mirror (somewhat) the [WHATWG working mode](https://whatwg.org/working-mode) and well as the [WebAppsWG working mode](https://www.w3.org/2022/04/webapps-wg-charter.html#working-mode), both of which work really well. 

We can add OWNERS as that automatically flags folks for review, or we can easily set that up via the GitHub settings (that someone with write access needs to also approve it, and we basically just agree on who needs to give a ✅.... it's a lightweight process).

Reply:
Filed https://github.com/WICG/digital-credentials/issues/60 for OWNERS.

Reply:
Re-opening this: didn't we agree to have co-editors (in addition to OWNERs) from other implementations on the last call?